.github/workflows/test.yml

```diff
@@ -22,4 +22,19 @@ jobs:
         env:
           AWS_ACCESS_KEY_ID: ${{ secrets.AWS_ACCESS_KEY_ID }}
           AWS_SECRET_ACCESS_KEY: ${{ secrets.AWS_SECRET_ACCESS_KEY }}
-        run: make test
\ No newline at end of file
+        run: make test
+
+      # GHA aws-cli issue https://github.com/aws/aws-cli/issues/5262#issuecomment-801301976
+      - name: Configure AWS Region
+        run: echo "AWS_DEFAULT_REGION=us-east-1" >> $GITHUB_ENV
```

Comment:
Why we can't create it as env var? We have some env vars down below

Reply:
that's to isolate the issue mentioned in the comment above